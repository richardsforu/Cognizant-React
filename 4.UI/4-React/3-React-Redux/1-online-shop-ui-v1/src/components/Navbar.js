// import React, { Component } from 'react';
/* 
class Navbar extends Component {
    render() {
        let { title } = this.props
        return (
            <div>
                <nav className="navbar navbar-light bg-primary">
                    <a className="navbar-brand" href="/">{title}</a>
                </nav>
            </div>
        );
    }
}

export default Navbar; */

// 
/* 

import React from 'react'
function Navbar(props) {
    let { title } = props
    return (
        <div>
            <nav className="navbar navbar-light bg-primary">
                <a className="navbar-brand" href="/">{title}</a>
            </nav>
        </div>
    )
}

export default Navbar;

*/

import React from 'react'
export default ({ title }) => {
    return (
        <div>
            <nav className="navbar navbar-light bg-primary">
                <a className="navbar-brand" href="/">{title}</a>
            </nav>
        </div>
    )
}