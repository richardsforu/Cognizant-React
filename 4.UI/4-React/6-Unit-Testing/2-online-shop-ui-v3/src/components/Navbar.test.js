import React from 'react';
import renderer from 'react-test-renderer'
import Navbar from './Navbar';

it('Renders movies', () => {
    const component = renderer.create(
        <Navbar title="shop-IT" />
    );
    expect(component).toMatchSnapshot();
});