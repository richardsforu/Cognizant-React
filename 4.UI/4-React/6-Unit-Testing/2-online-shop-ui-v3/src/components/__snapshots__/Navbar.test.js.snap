// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders movies 1`] = `
<div>
  <nav
    className="navbar navbar-light bg-primary"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      shop-IT
    </a>
  </nav>
</div>
`;
